EIPS/eip-5252.md

```diff
@@ -0,0 +1,187 @@
+---
+eip: 5252
+title: Account-bound Finance
+description: An EIP-5114 extension that aids in preventing loss of funds
+author: Hyungsuk Kang (@hskang9)
+discussions-to: https://ethereum-magicians.org/t/pr-5252-discussion-account-bound-finance/10027
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-06-29
+requires: 721, 1155, 5114
+---
+
+## Abstract
+This EIP proposes a form of smart contract design pattern and a new type of account abstraction on how one's finance should be managed in web3.0, ensuring transparency of managing investments and protection with self-sovereignty even from its financial operators.
+
+This EIP extends [EIP-5114](./eip-5114.md), but the difference is that it can be transferred to other accounts for mobility between managing multiple wallets.
+
+## Motivation
+
+Finance in the crypto ecosystem is facing a huge trust issue. Smart contracts are often proxies, with the actual logic of the contract hidden away in a separate logic contract. Many projects include a multi-signature "wallet" with unnecessarily-powerful permissions. And it is not possible to independently verify that stablecoins have enough real-world assets to continue maintaining their peg, creating a large loss of funds (such as happened in the official bankruptcy announcement of Celsius and UST de-pegging and anchor protocol failure). One should not trust exchanges or other third parties with one's own investments.
+
+The pattern empowers more self-sovereignty and gives more credentials(obviously with an Account-bound token) than locking financial data in the operating team's contract.
+
+Segregation between investor's fund and operation fee is clearly specified in the smart contract, so investors can ensure safety from arbitrary loss of funds by the operating team's control.
+
+#### Offchain-identity vs Soul-bound token on credentials 
+
+This EIP opposes the offchain-identity solution to take over the whole system, because their backend eventually relies on the trust of the operator, not the cryptographic proof of storing data(e.g. Proof-of-work, Proof-of-stake, etc). Offchain identity as credentials denies the whole premise of crypto's existence. Hence, Soul-bound token is the verifiable credential to stay, and offchain integration is just enough to show images.
+
+## Specification
+
+The specification consists of two patterns for **Interaction** and **Governance**.
```

Comment:
Can you add basic specification of what is being required in this EIP?

Reply:
I added requirements for properties. If there is a need for function with solidity interface, please let me know.